aiosmtpd/context.py

```diff
@@ -0,0 +1,15 @@
+class SessionContext:
```

Comment:
What do you think about calling this class just `Session`?  I'm not sure the `Context` suffix adds much additional information.

Have you seen the [attrs](https://attrs.readthedocs.io/en/stable/) package?  It might be overkill for what you're adding here (and additional dependencies should always be considered carefully), but if it's helpful I'd be okay with it.  I've been using attrs in other projects and it's really quite nice.